### ISSUE TITLE
Accessibility Testing

### ISSUE THREAD

**Issue by thekodester** [wet-boew/wet-boew#8415]
This is just a heads up for anyone testing accessibility with Firefox. Currently VoiceOver on macOS doesn't work with Firefox since the update. Not sure if others are affected.

See https://bugzilla.mozilla.org/show_bug.cgi?id=1466342 .
